Misc/NEWS.d/next/Library/2025-05-25-23-23-05.gh-issue-134151.13Wwsb.rst

```diff
@@ -0,0 +1,2 @@
+:mod:`email`: Fix TypeError in email.utils.decode_params() when sorting RFC
+2231 continuations on Python 3.
```

Comment:
"...continuations that contain an unnumbered section."  I don't think we need to say "on Python 3" at this point.